src/routes.js

```diff
@@ -0,0 +1,14 @@
+const usersControllers = require('./controllers/users')
+
+const router = require('express').Router()
+
+// user routes
+router.route('/users')
+  .get(usersControllers.get)
+  .post(usersControllers.post)
```

Comment:
Call this one `userControllers.create`.

If you feel strongly about that do not worry